src/main/java/com/endava/soa_5/basic/Exercise_1.java

```diff
@@ -0,0 +1,36 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_1 {
```

Comment:
Rename please the class to be more representative of what is done here. Do this for all your classes that have a similar name

Reply:
Done